content/styles/02-molecules/preview.css

```diff
@@ -0,0 +1,34 @@
+.preview {
+  margin: 0;
+}
+
+.preview a {
+  color: var(--black);
+}
+
+.preview a:hover figure::before {
+  color: var(--secondary-color);
+  font-size: 8rem;
+  bottom: -0.5rem;
```

Comment:
I would put the bottom… on top ^^
bottom being a position property should be above appearance

Reply:
totally true